app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,32 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('System Images');
+  },
+  model() {
+    return ({
+      topics: [
+        {
+          id   : 1,
+          name : 'Automobiles'
+        },
+        {
+          id   : 2,
+          name : 'Yoga'
+        },
+        {
+          id   : 3,
+          name : 'charity'
+
+        },
+        {
+          id   : 4,
+          name : 'Travel'
+        }
+      ]
+    });
+  },
+  afterModel(model) {
+    this._super(...arguments);
+    this.transitionTo('admin.content.system-images.list', model.topics[0].id);
```

Comment:
This transition should be done only if the user is attempting to goto `/admin/content/system-images`. And it shouldn't be done when the user goes to `/admin/content/system-images/<id>`.

And also, use `replaceWith` instead of `transitionTo`